app/mailers/article_mailer.rb

```diff
@@ -0,0 +1,21 @@
+class ArticleMailer < ApplicationMailer
+  default from: 'support@ckblogs.com'
+
+  def create_mail(object)
+    @object = object
+    @user = params[:user]
+    mail to: @user.email, subject: "A new entry for #{object.class} has been created"
+  end
+
+  def update_mail(object)
+    @object = object
+    @user = params[:user]
+    mail to: @user.email, subject: "A new entry for #{object.class} has been updated"
+  end
+
+  def delete_mail(object)
+    @object = object
+    @user = params[:user]
+    mail to: @user.email, subject: "A new entry for #{object.class} has been deleted"
+  end
+end
```

Comment:
we don't need that as of now, you can keep this locally so that will use later if needed